creds.py

```diff
@@ -1,9 +1,11 @@
+# To use the scraper, you have to replace the credentials below with real values
+
 # ais.usvisa-info.com credentials
 username = 'your_username'
 password = 'your_strong_password'
 url_id = '41231513' # This is the number id in the url (e.g.: https://ais.usvisa-info.com/en-pe/niv/schedule/41231513)
 country_code = 'pe' # This is the country code in the url
 
-# Telegram bot token and chat 
+# Telegram bot token and chat
```

Comment:
I suspect this change is because of a line break? Can you confirm please?

Reply:
It's trailing whitespace that gets removed